x-pack/plugins/fleet/server/config.ts

```diff
@@ -174,6 +174,7 @@ export const config: PluginConfigDescriptor = {
       })
     ),
     enabled: schema.boolean({ defaultValue: true }),
+    createArtifactsBulkBatchSize: schema.maybe(schema.number()),
```

Comment:
Also, can you add some info. about what this number represents?  Initially, I thought it was the number of items in a batch, but looks like really it should be the number of bytes per batch (right?)

Reply:
I thought we decided to not rely on the number of items as there can be items with small body and others with large one. It is tied to the `encoded_size` of an artifact. So the sum of the encoded_size of artifacts in a batch shouldn't be bigger than this value.
Agree on the default and max/min values, will apply that.

Reply:
right. Sorry if I confused you. I mean: document what this number represents so that its clear what it should be set to. (ex. its the number of bytes)